journal/frontend/src/features/storage/local.ts

```diff
@@ -35,7 +35,8 @@ export function useLocalStorage<TValue = string>(
     }
 
     const cleanupHandlers = receiveStorageChangeEvents(key, (value) => {
-      onLocalStorageChange(parseJSON(value))
+      const unSerialized = typeof value === 'string' ? parseJSON(value) : value
```

Comment:
This is awkward. How is it that we ever read anything out of local storage that's not a string? Is this to handle null?

Reply:
This was the part that was super confusing for me. It is triggered from the UI move card event and the value being changed is an array of IDs post sort that is eventually being stored in local storage.